common/crc32_generic.h

```diff
@@ -0,0 +1,98 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2025 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+constexpr unsigned int crc32_table[256] =
```

Comment:
Maybe use `std::array` in such cases?

Reply:
I've decided not to do modification apart from changing `const` on `constexpr`